ui/apps/platform/src/Components/CompoundSearchFilter/types.ts

```diff
@@ -510,6 +510,60 @@ export type NodeComponentAttributeInputType = ValueOf<
     NodeComponentSearchFilterConfig['attributes']
 >['inputType'];
 
+// Profile Check search filter
+
+export const profileCheckSearchFilterConfig = {
+    displayName: 'Profile Check',
+    searchCategory: 'COMPLIANCE',
```

Comment:
Need to update to the appropriate searchCategory for Compliance 2.0 (same for the others below)